polynote-runtime/src/main/scala/polynote/runtime/python/PythonObject.scala

```diff
@@ -117,33 +117,38 @@ object PythonObject {
           }
       }
 
-      val basicMimeReprs = Map(
-        "text/html"           -> "_repr_html_",
-        "text/plain"          -> "__repr__",
-        "application/x-latex" -> "_repr_latex_",
-        "image/svg+xml"       -> "_repr_svg_",
-        "image/jpeg"          -> "_repr_jpeg_",
-        "image/png"           -> "_repr_png_"
-      ).flatMap {
-        case (mime, funcName) =>
-          if (obj.hasAttribute(funcName)) attemptRepr(mime, obj.applyDynamic(funcName)()) else None
-      }
+      if (obj.hasAttribute("__init__")) { // For now, don't bother with classes (in future we might want to do something special?)
```

Comment:
This isn't really part of this PR, but I defined a class as part of my testing and it kept failing to call __repr__ on the class very noisily. Not sure why it was being noisy here, but figured it made sense to short-circuit if we know it's a class anyways. 

Checking for `__init__` to determine whether it's a class seems to make sense, but I'm not sure if there's a better attribute to check instead.

Reply:
This didn't work 🤦‍♂ Changed it to something more correct (but jankier?)